python-avd/pyavd/_cv/api/arista/alert/v1/__init__.py

```diff
@@ -7,28 +7,16 @@
 # This file has been @generated
 
 from dataclasses import dataclass
-from datetime import (
-    datetime,
-    timedelta,
-)
-from typing import (
-    TYPE_CHECKING,
-    AsyncIterator,
-    Dict,
-    List,
-    Optional,
-)
+from datetime import datetime, timedelta
```

Comment:
do we want any change under `_cv/api`? Maybe we should revert them to avoid conflicts the next time around?

Reply:
I think it happened with some isort stuff. I will revert

Reply:
The changes I have not comes from pre-commit. Not sure how they were not applied last time.

Reply:
Hmm I think they should be ignored by isort from our config no?
```
[tool.isort]
extend_skip_glob = [
  "python-avd/pyavd/_cv/api/**/*"
]
```

Reply:
They should yes. But they are not (on my machine).